src/Security/MemberAuthenticator/Authenticator.php

```diff
@@ -0,0 +1,193 @@
+<?php
+
+namespace SilverStripe\Security\MemberAuthenticator;
+
+use SilverStripe\Control\Controller;
+use SilverStripe\Control\Session;
+use SilverStripe\ORM\ValidationResult;
+use InvalidArgumentException;
+use SilverStripe\Security\Authenticator as BaseAuthenticator;
+use SilverStripe\Security\Security;
+use SilverStripe\Security\Member;
+use SilverStripe\Security\LoginAttempt;
+
+/**
+ * Authenticator for the default "member" method
+ *
+ * @author Markus Lanthaler <markus@silverstripe.com>
+ */
+class Authenticator implements BaseAuthenticator
+{
+
+    public function supportedServices()
+    {
+        // Bitwise-OR of all the supported services, to make a bitmask
```

Comment:
It's a matter of how you read it. I read it as "All the supported services _in this authenticator_ "